src/main/java/com/google/api/codegen/metadatagen/PackageMetadataGeneratorTool.java

```diff
@@ -0,0 +1,173 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.metadatagen;
+
+import com.google.api.codegen.metadatagen.py.PythonPackageCopier;
+import com.google.api.tools.framework.tools.ToolOptions;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+/** Tool to generate package metadata. */
+public class PackageMetadataGeneratorTool {
+
+  private static final Map<String, List<String>> SNIPPETS =
+      new ImmutableMap.Builder<String, List<String>>()
+          .put(
+              "python",
+              Lists.newArrayList(
+                  "py/setup.py.snip",
+                  "py/README.rst.snip",
+                  "py/PUBLISHING.rst.snip",
+                  "py/MANIFEST.in.snip"))
+          .build();
+
+  private static final Map<String, PackageCopier> COPIERS =
+      new ImmutableMap.Builder<String, PackageCopier>()
+          .put("python", new PythonPackageCopier())
+          .build();
+
+  public static void main(String[] args) throws Exception {
+    Options options = new Options();
+    options.addOption("h", "help", false, "show usage");
+    options.addOption(
+        Option.builder()
+            .longOpt("descriptor_set")
```

Comment:
This is still needed to build on top of API Compiler, even if we don't yet use the `Model` that the API Compiler framework provides yet. It is easier to include the scaffolding in this initial PR than to have to refactor the tool in a follow-up, so I would like to leave this in.